common/config/rush/nonbrowser-approved-packages.json

```diff
@@ -86,6 +86,10 @@
       "allowedCategories": [ "libraries" ]
     },
     {
+      "name": "babel-jest",
```

Comment:
> "name": "babel-jest", [](start = 5, length = 22)

Let's eliminate this

Reply:
@nickpape-msft Qinwei said you introduced a test that relies on ES6 JavaScript?  Why do we need that?

---
In reply to: [145563051](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r145563051) [](ancestors = 145563051)

Reply:
Removed.

---
In reply to: [145563393](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r145563393) [](ancestors = 145563393,145563051)